File: Python/02. if문/2480_주사위 세개.py
dice1, dice2, dice3 = map(int, input().split())

prize = 0
if dice1 == dice2 == dice3:
    prize = 10000 + dice1 * 1000
elif dice1 == dice2 or dice1 == dice3:
    prize = 1000 + dice1 * 100
elif dice2 == dice3:
    prize = 1000 + dice2 * 100
else:
    prize = max(dice1, dice2, dice3) * 100

print(prize)

File: C++/02. if문/2480_주사위 세개.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
    int dice1, dice2, dice3, prize = 0;
    cin >> dice1 >> dice2 >> dice3;

    if (dice1 == dice2 && dice2 == dice3)
    {
        prize = 10000 + dice1 * 1000;
    } else if (dice1 == dice2 || dice1 == dice3)
    {
        prize = 1000 + dice1 * 100;
    } else if (dice2 == dice3)
    {
        prize = 1000 + dice2 * 100;
    } else
    {
        prize = max({dice1, dice2, dice3}) * 100;
    }

    cout << prize << endl;

    return 0;
}
